ewc-rfc002-present-verifiable-credentials.md

```diff
@@ -75,7 +75,7 @@ Authorisation requests can be presented to the wallet by verifying in two ways:
 ```sh
 openid4vp://?client_id=https://example.verifier.com
 &response_type=vp_token
-&scope=openid
+&scope=com.example.passport_credential_presentation
```

Comment:
Can a verifier use `scope` in conjunction with the `presentation_definition` field?

Reply:
I did not find anything which would not allow this approach. Only this sentence: presentation_definition MUST be present when presentation_definition_uri parameter, or a scope value representing a Presentation Definition is not present.
But if it makes sense to have both in there is questionable.